code/modules/events/slaughterevent.dm

```diff
@@ -0,0 +1,69 @@
+
```

Comment:
I tested the event version of it, but not the antag spawner version had trouble calling that proc.